app/views/subscriptions/confirm_delete_all.erb

```diff
@@ -0,0 +1,23 @@
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading">
+  <% if @subscribable_type %>
+    <%= t(".page_heading.#{@subscribable_type}") %>
+  <% else %>
+    <%= t(".page_heading.all") %>
+  <% end %>
+</h2>
+
+<!--main content-->
+<%= form_for(@user, :url => delete_all_user_subscriptions_path(@user, type: @subscribable_type), :html => {:method => :post, :class => "simple destroy"}) do |f| %>
```

Comment:
Could you update the code style here to use JSON style in place of the hash rockets?